src/v3/src/components/Form/Form.tsx

```diff
@@ -92,9 +95,8 @@ const Form: FunctionComponent<{
     <form
       noValidate
       onSubmit={handleSubmit}
-      className="o-form" // FIXME update page objects using .o-form selectors
+      className={classes} // FIXME update page objects using .o-form selectors
```

Comment:
I just made this JIRA for tech debt, can you attach the ticket number? https://oktainc.atlassian.net/browse/OKTA-578584